cogs/csgo.py

```diff
@@ -375,6 +410,7 @@ async def pug(self, ctx: commands.Context, *args):
                     await player.send(embed=connect_embed)
                 except discord.HTTPException or discord.Forbidden:
```

Comment:
```suggestion
                except (discord.HTTPException, discord.Forbidden):
```
Incorrect syntax for except